src/test/java/org/rulez/demokracia/pdengine/BeatTableCompareBeatsTest.java

```diff
@@ -0,0 +1,70 @@
+package org.rulez.demokracia.pdengine;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.rulez.demokracia.pdengine.annotations.TestedBehaviour;
+import org.rulez.demokracia.pdengine.annotations.TestedFeature;
+import org.rulez.demokracia.pdengine.annotations.TestedOperation;
+import org.rulez.demokracia.pdengine.exception.ReportedException;
+import org.rulez.demokracia.pdengine.testhelpers.CreatedBeatTable;
+
+@TestedFeature("Schulze method")
+@TestedOperation("compare beats")
+public class BeatTableCompareBeatsTest extends CreatedBeatTable {
+
+	@Before
+	public void setUp() throws ReportedException {
+		super.setUp();
+	}
+
+	@TestedBehaviour("if beats are different, the bigger wins")
```

Comment:
You should give this behaviour its own test class, as it is tested by more than 3 tests, and this makes PMD nervous.